make/mt-os/mt-cc3200.mk

```diff
@@ -5,41 +5,59 @@
 
 include make/mt-os/mt-os-common.mk
 
-###
-## COMPILER NAME
-###
-COMPILER ?= arm_15.12.3.LTS
+ifdef XI_TRAVIS_BUILD
+	# For Xively C Client Continuous Integration only.
+	COMPILER ?= arm_15.12.3.LTS
```

Comment:
I guess it is "just" adding the new path (similar to xively-client-artifactory/ti/ccsv6/tools/compiler/arm_15.12.3.LTS) to the artifactory. Although it has the be the linux version of the ccsv7.